packages/fluent/scss/dataviz/_layout.scss

```diff
@@ -475,22 +455,52 @@
         series-30: $kendo-series-30,
 
         gauge-pointer: $kendo-series-f,
-        gauge-track: if($kendo-enable-color-system, k-color( base-emphasis ), k-get-theme-color-var( neutral-30 ))
+        gauge-track: if($kendo-enable-color-system, k-color( base-emphasis ), k-get-theme-color-var( neutral-30 )),
+
+        chart-font-size: $kendo-chart-font-size,
+        chart-title-font-size: $kendo-chart-title-font-size,
+        chart-pane-title-font-size: $kendo-chart-pane-title-font-size,
+        chart-pane-title-font-weight: $kendo-chart-pane-title-font-weight,
+        chart-label-font-size: $kendo-chart-label-font-size,
+
+        chart-inactive: $kendo-chart-inactive-bg,
+        chart-major-lines: $kendo-chart-major-lines,
+        chart-minor-lines: $kendo-chart-minor-lines,
+        chart-area-opacity: $kendo-chart-area-opacity,
+        chart-area-inactive-opacity: $kendo-chart-area-inactive-opacity,
+        chart-line-inactive-opacity: $kendo-chart-line-inactive-opacity,
+        chart-notes-background: $kendo-chart-notes-background,
+        chart-notes-border: $kendo-chart-notes-border,
+        chart-notes-lines: $kendo-chart-notes-lines,
+        chart-crosshair-background: $kendo-chart-crosshair-background,
+        chart-error-bars-background: $kendo-color-error-bars-background,
+
     );
 
+    :root {
+        @each $name, $value in $exported {
+            --kendo-chart-#{$name}: #{$value};
+        }
+    }
+
     @each $name, $value in $exported {
         $type: meta.type-of($value);
 
         .k-var--#{$name} {
-            @if $type == "color" {
+            @if $type == "color" or $type == "string" and string.index($value, "color") {
                 // background-color can store any color
-                background-color: $value;
-            } @else if $type == "string" and string.slice( $value, 1, 4 ) == "var(" {
-                background-color: $value;
+                background-color: var(--kendo-chart-#{$name});
+            } @else if string.index($name, "font-size") {
+                font-size: var(--kendo-chart-#{$name});
+            } @else if string.index($name, "font-weight") {
+                font-weight: var(--kendo-chart-#{$name});
+            } @else if string.index($name, "opacity") {
+                opacity: var(--kendo-chart-#{$name});
             } @else if $type == "number" {
                 // margin-top can store positive & negative values
-                margin-top: $value;
+                margin-top: var(--kendo-chart-#{$name});
```

Comment:
Is the margin-top used at all? I don't see any margins exported in the table, so I think we can delete this as well (we can still double check with the Dataviz team)